Automattic/jetpack
# Launchpad: Update function prefixes in task definitions

## Proposed changes:

* Update the majority of functions in `launchpad-task-definitions.php` to use the same standard prefix, `wpcom_launchpad_`, to avoid potential conflicts down the road.
* The majority of these callbacks are self-contained in this file; a couple have instances in `launchpad.php`.
* This PR does *not* update functions that are also called outside `jetpack-mu-wpcom` to avoid breakage. I've noted instances of these functions and their locations in the doc blocks. These will need to be handled differently:

`wpcom_mark_launchpad_task_complete`
`wpcom_track_site_launch_task`
`wpcom_trigger_email_campaign`


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this PR to your sandbox and sandbox public-api
* Create new sites for each of the types of Launchpads, both pre- and post-launch, from `/setup` unless otherwise specified:

o Free (Choose a free plan during onboarding)
o Build (Choose "Promote myself or business" during onboarding)
o Write (Choose "Write and publish" during onboarding)
o Free Newsletter (Start from `/setup/newsletter`, choose free)
o Paid Newsletter (Start from `/setup/newsletter`, choose paid)
o Newsletter with subscribers (Start from `/setup/newsletter`, choose import existing subscribers)
o Link in bio (Start from `/link-in-bio`)
o VideoPress (Start from `/setup/videopress`)

* Ensure you can still complete all tasks for each list.
